### ISSUE TITLE
DESeq2 install

### ISSUE THREAD

**Issue by HadrienG** [ngs-docs/angus#290]
On a new instance, following [http://angus.readthedocs.io/en/2017/deseq2-asthma.html](http://angus.readthedocs.io/en/2017/deseq2-asthma.html), `install-deseq2.R` doesn't install GenomicFeatures (nor DESeq2)

```$ R
R version 3.4.1 (2017-06-30) -- "Single Candle"
```

```
> library(GenomicFeatures)
Error in library(GenomicFeatures) :
  there is no package called ‘GenomicFeatures’
```

when installing with `install-deseq2.R`:

```ERROR: dependencies ‘GenomeInfoDb’, ‘GenomicRanges’, ‘SummarizedExperiment’, ‘Rsamtools’ are not available for package ‘GenomicAlignments’
* removing ‘/usr/local/lib/R/site-library/GenomicAlignments’
ERROR: dependency ‘annotate’ is not available for package ‘genefilter’
* removing ‘/usr/local/lib/R/site-library/genefilter’
ERROR: dependency ‘annotate’ is not available for package ‘geneplotter’
* removing ‘/usr/local/lib/R/site-library/geneplotter’
ERROR: dependencies ‘GenomicRanges’, ‘XML’, ‘GenomeInfoDb’, ‘RCurl’, ‘Rsamtools’, ‘GenomicAlignments’ are not available for package ‘rtracklayer’
* removing ‘/usr/local/lib/R/site-library/rtracklayer’
ERROR: dependencies ‘GenomicRanges’, ‘SummarizedExperiment’, ‘genefilter’, ‘geneplotter’ are not available for package ‘DESeq2’
* removing ‘/usr/local/lib/R/site-library/DESeq2’
ERROR: dependencies ‘GenomeInfoDb’, ‘GenomicRanges’, ‘RCurl’, ‘rtracklayer’, ‘biomaRt’ are not available for package ‘GenomicFeatures’
* removing ‘/usr/local/lib/R/site-library/GenomicFeatures’```

**Comment by HadrienG** [comment]
Rcurl might be the problem

```
installing *source* package ‘RCurl’ ...
** package ‘RCurl’ successfully unpacked and MD5 sums checked
checking for curl-config... no
Cannot find curl-config
```

**Comment by bluegenes** [comment]
upendra solved this -- I'll grab the code

**Comment by bluegenes** [comment]
```
sudo apt-get install libxml2 libxml2-dev libcurl4-gnutls-dev libssl-dev -y
Rscript -e 'source("http://bioconductor.org/biocLite.R"); biocLite("DESeq2");
```

**Comment by HadrienG** [comment]
Closed with f468ad9
